rmrk-token/io/src/lib.rs

```diff
@@ -0,0 +1,548 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use resource_io::Resource;
+use scale_info::TypeInfo;
```

Comment:
```suggestion
use gstd::{prelude::*, ActorId};
use resource_io::Resource;
```
These traits have already been imported by `gstd::prelude::*`.